### ISSUE TITLE
Audio-Separator-UI

### ISSUE THREAD

**Issue by Bebra777228** [nomadkaraoke/python-audio-separator#164]
**Repository Link:** [Audio-Separator-UI](https://github.com/Bebra777228/Audio-Separator-UI)
**PyPI Link:** [PyPI Audio-Separator-UI](https://pypi.org/project/audio-separator-ui/)
I also made minor changes to the interface and described them in the [README](https://github.com/Bebra777228/Audio-Separator-UI?tab=readme-ov-file#gradio-interface).

I have also sent you a collaborator invitation. You can accept it and make any necessary changes to my repository.

Regarding the package, I decided to synchronize the versions. You have 0.25.0, and so do I. If you have an update, I will update mine as well. The Gradio interface does not require significant changes, and all modifications occur within a single file. Therefore, I believe this is a reasonable approach. Given that the interface is already ready for use and there is practically nothing to change, the only thing left is to keep this repository in sync with yours.

And I hope I did everything right :)

**Comment by beveradb** [comment]
Hey @Bebra777228 - this is your choice (as it's your repo/project) but I would encourage you to change your code for `Audio-Separator-UI` so it's just including `audio-separator` as a python dependency, not copying / duplicating all of the code from this repo 😅 

That way there's a lot less code to maintain in that repo, you don't need to be constantly pulling in any changes from this repo into your own, etc. - it's also generally best practice to avoid duplicating code wherever possible. 

I know I'm a bit of a hypocrite as most of the model inferencing code in this repo comes directly from the UVR codebase, but there was no way for me to use that as a library (which is actually why I built this project!) whereas `audio-separator` is 100% designed to be useful as a library 😄 

Hope that makes sense - if you're not sure how to actually do this, I can potentially raise a PR on your repo showing what I mean but it'll be a pretty big PR deleting most of the code except the gradio UI stuff!

**Comment by Bebra777228** [comment]
Yes, I have a general idea of how to implement this, but if it's not too much trouble, it would be great if you could raise a PR. I might have misunderstood some parts, so your guidance would be really helpful.
